package.json

```diff
@@ -250,7 +251,8 @@
   "templateOSS": {
     "rootRepo": false,
     "rootModule": false,
-    "version": "3.7.1"
+    "version": "3.8.0",
+    "releaseTest": "release.yml"
```

Comment:
This is a template-oss escape hatch I added so `release-please.yml` can do a `workflow_call` to any other action. The root cli package isn't under template-oss control so we manage this file ourselves and let the template-oss controlled `release-please.yml` call it.